README.md

```diff
@@ -1,4 +1,174 @@
-### Running notebooks
+## API
+
+#### convert_from_process_graph
+
+```python
+convert_from_process_graph(
+    process_graph,
+    n_output_bands=1,
+    sample_type="FLOAT32",
+    units=None,
+    bands_dimension_name="bands",
+    temporal_dimension_name="t"
+)
+```
+
+###### Parameters
+
+* **`process_graph`**: *dict*
+
+    Process graph JSON as Python dict object.
```

Comment:
Maybe "...**openEO** process graph JSON..." so it's even clearer. If you agree this should be changed everywhere process graph is used.